scripts/fetch-secrets.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+if [ "${BASH_SOURCE-}" = "$0" ]; then
+    echo "You must source this script: \$ source $0" >&2
+    exit 33
+fi
+
+THIEF_KEYVAULT_NAME="thief-kv"
+
+function get-secret {
+    bash_name=$1
+    kv_name=$2
+    echo "Fetching ${bash_name}"
+    value=$(az keyvault secret show --vault-name ${THIEF_KEYVAULT_NAME} --name ${kv_name} | jq -r ".value")
```

Comment:
for my sanity: how is this being run? I assume locally, so you're already logged into your azure account via the CLI?

Reply:
Worth a comment about how to set up an environment so this runs.

Reply:
Yes, this gets called manually.  Docs coming.

---
In reply to: [491169089](https://github.com/Azure/iot-sdk-longhaul/pull/9#discussion_r491169089) [](ancestors = 491169089)

Reply:
doc is coming in my next review.

---
In reply to: [492340762](https://github.com/Azure/iot-sdk-longhaul/pull/9#discussion_r492340762) [](ancestors = 492340762)